plugins/unocss.ts

```diff
@@ -0,0 +1,188 @@
+   */
+  config?: UserConfig;
+  /**
+   * Enable AOT mode - run UnoCSS to extract styles during the build task.
+   * Enabled by default.
+   */
+  aot?: boolean;
+  /**
+   * Enable SSR mode - run UnoCSS live to extract styles during server renders.
+   * Disabled by default.
+   */
+  ssr?: boolean;
+  /**
+   * Enable CSR mode - run the UnoCSS runtime on the client.
+   * It will generate styles live in response to DOM events.
+   * Disabled by default.
+   */
+  csr?: boolean;
+};
+
+/**
+ * Helper function for typing of config objects
+ */
+export function defineConfig<T extends object = Theme>(config: UserConfig<T>) {
+  return config;
+}
+
+/**
+ * Installs a hook in Preact to extract classes during server-side renders
+ * @param classes - Set of class strings, which will be mutated by this function.
+ */
+export function installPreactHook(classes: Set<string>) {
+  // Hook into options._b which is called whenever a new comparison
+  // starts in Preact.
+  const originalHook = (preactOptions as PreactOptions).__b;
+  (preactOptions as PreactOptions).__b = (
+    // deno-lint-ignore no-explicit-any
+    vnode: VNode<JSX.DOMAttributes<any>>,
+  ) => {
+    if (typeof vnode.type === "string" && typeof vnode.props === "object") {
+      const { props } = vnode;
+      if (props.class) {
+        props.class.split(" ").forEach((cls) => classes.add(cls));
+      }
+      if (props.className) {
+        props.className.split(" ").forEach((cls) => classes.add(cls));
+      }
+    }
+
+    originalHook?.(vnode);
+  };
+}
+
+/** UnoCSS plugin - automatically generates CSS utility classes */
+export default function unocss(
+  { config, aot = true, ssr = false, csr = false }: UnoCssPluginOptions = {},
+): Plugin {
+  // A uno.config.ts file is required in the project directory if a config object is not provided,
+  // or to use the browser runtime
+  const configURL = new URL("./uno.config.ts", Deno.mainModule);
+
+  // Load config from file if required
+  if (config === undefined) {
+    import(configURL.toString()).then((mod) => {
+      config = mod.default;
+    }).catch((error) => {
+      exists(configURL, { isFile: true, isReadable: true }).then(
+        (configFileExists) => {
+          throw configFileExists ? error : new Error(
+            "uno.config.ts not found in the project directory! Please create it or pass a config object to the UnoCSS plugin",
+          );
+        },
+      );
+    });
+  }
+
+  return {
+    name: "unocss",
+    // Optional client runtime
+    entrypoints: csr
+      ? {
+        "main": `
+        data:application/javascript,
+        import config from "${configURL}";
+        import init from "https://esm.sh/@unocss/runtime@0.56.5";
+        export default function() {
+          window.__unocss = config;
+          init();
+        }`,
+      }
+      : {},
+
+    async renderAsync(ctx) {
+      // Link to CSS file, if AOT mode is enabled
+      const cssLinks = aot ? [{ url: "/uno.css" }] : [];
```

Comment:
Line 131 should be updated to `const links = aot ? [{ rel: "stylesheet", href: "/uno.css" }] : [];` to match the new `links` and also the new Link format